code/modules/mob/living/carbon/damage_procs.dm

```diff
@@ -114,13 +114,13 @@
  * * slot - organ slot, like [ORGAN_SLOT_HEART]
  * * amount - damage to be done
  * * maximum - currently an arbitrarily large number, can be set so as to limit damage
- * * required_organtype - targets only a specific organ type if set to ORGAN_ORGANIC or ORGAN_ROBOTIC
+ * * required_organ_flag - targets only a specific organ type if set to ORGAN_ORGANIC or ORGAN_ROBOTIC
  */
-/mob/living/carbon/adjustOrganLoss(slot, amount, maximum, required_organtype)
+/mob/living/carbon/adjustOrganLoss(slot, amount, maximum = INFINITY, required_organ_flag = NONE)
```

Comment:
Assuming INFINITE is so that people don't get no effect when they don't set a maximum?

Reply:
Yes.